idd/Energy+.idd.in

```diff
@@ -39703,56 +39739,61 @@ AirTerminal:SingleDuct:ConstantVolume:CooledBeam,
        \minimum> 0.0
        \default 0.0145
 
-AirTerminal:SingleDuct:InletSideMixer,
-       \memo The inlet side mixer air terminal unit provides a means of supplying central system air
-       \memo to the air inlet of a zone AC unit such as a four pipe fan coil. Normally the central air
-       \memo would be ventilation air from a dedicated outdoor air system (DOAS).
+AirTerminal:SingleDuct:Mixer,
+       \memo The mixer air terminal unit provides a means of supplying central system
+       \memo air to the air inlet or outlet side of a zoneHAC equipment such as a four
+       \memo pipe fan coil unit. Normally the central air would be ventilation air from
+       \memo a dedicated outdoor air system (DOAS).
    A1, \field Name
        \required-field
    A2, \field ZoneHVAC Terminal Unit Object Type
        \required-field
        \type choice
        \key ZoneHVAC:FourPipeFanCoil
        \key ZoneHVAC:WaterToAirHeatPump
+       \key ZoneHVAC:PackagedTerminalAirConditioner
+       \key ZoneHVAC:PackagedTerminalHeatPump
+       \key ZoneHVAC:TerminalUnit:VariableRefrigerantFlow
+       \key AirLoopHVAC:UnitarySystem
+       \note The type of ZoneHVAC equipment to which this terminal mixer will be connected.   
    A3, \field ZoneHVAC Terminal Unit Name
        \required-field
        \type object-list
        \object-list DOAToZonalUnit
+       \note The name of ZoneHVAC equipment to which this terminal mixer will be connected.   
    A4, \field Terminal Unit Outlet Node Name
```

Comment:
@Nigusse And I might consider changing the three "Terminal Unit . . . Node Name" fields to be "Mixer . . . Node Name"